root/components/Home/AppDownload.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const AppDownload = (): React.Element<typeof section> => {
+  return (
+    <section className="section cta-section">
+      <div className="container">
+        <div className="row align-items-center">
+          <div
+            className="col-md-6 me-auto text-center
+             text-md-start mb-5 mb-md-0"
+          >
+            <h2>
+              {l(`Download and Use our App and Software`)}
```

Comment:
I'd argue that while that's true re apps/software, many people will read "app is for phone, software is for PC". That said, we already list the platforms below so it might not matter, but then I'd say "Download our software" rather than "Download our app" because to me calling PC software apps is weirder than calling Android apps software :)

I agree with dropping "use" and probably would use lowercase but then I like lowercase everywhere.

Reply:
Done

Reply:
> many people will read "app is for phone, software is for PC"

That's why I suggested app actually, because the section seemed to primarily be for the phone apps.  But I guess your point about "calling PC software apps is weird" is the opposite on macOS, where everything is called an app.

Reply:
I kinda agree that an app is more commonly used these days. As @reosarevok said, I was planning to separate the app and the Picard software previously. But @mwiencek has a good point that an app is the standard naming convention these days since it's anyway a short form for `Application`. Let's just go with something like `Download our apps` then? (`apps` because we want the user to have the app on their phone and Picard on their desktop/mac)

Reply:
I believe we should have a separate page for the affiliated taggers with some descriptions for them there.
I'm fine with removing Picard from this section because we are going to anyway highlight it in the footer and the projects section!